### PR TITLE
Merge release to master

### PR DESCRIPTION
**Pull request by ghale**
Resolves conflicts and merges the latest changes on release to master.

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -77,31 +77,6 @@ ADD RELEASE FEATURES ABOVE
...
-```groovy
-beforeSettings { settings ->
-    settings.caches {
-        cleanup = Cleanup.DISABLED
-    }
-}
-```
-
-See [Configuring cleanup of caches and distributions](userguide/directory_layout.html#dir:gradle_user_home:configure_cache_cleanup) for more information.
-
```
Comment (ghale) [inline subprojects/docs/src/docs/release/notes.md:104]: This was left over in the release notes from 8.0 and didn't get removed when the 8.1 notes were cut.  Just happened to notice it while resolving the conflict.

Comment (hythloda) [reply subprojects/docs/src/docs/release/notes.md:104]: good catch

**Review by hythloda — APPROVED** [review]

**Review by ljacomet — APPROVED** [review]

**Comment by ghale** [comment]
@bot-gradle test and merge

**Comment by ghale** [comment]
@bot-gradle test and merge

**Pull request merged**